sdk/python/feast/permissions/permission.py

```diff
@@ -196,7 +206,10 @@ def to_proto(self) -> PermissionProto:
             name_pattern=self.name_pattern if self.name_pattern is not None else "",
             actions=actions,
             policy=self.policy.to_proto(),
-            tags=self._tags if self._tags is not None else None,
+            tags=self.tags if self.tags is not None else None,
```

Comment:
isn't this equivalent to 
```py
tags=self.tags
required_tags=self.required_tags
```
In fact, when they are `None` the if command returns `None`